.gitignore

```diff
@@ -129,3 +129,6 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+#
```

Comment:
Comment could be temporary mac os  files

Reply:
I second that. As someone with linux blindness (only really using linux for a while and so a bit dumb with other oses) I'm not sure what is being ignored here without a comment.